projects/aca-content/src/lib/components/bulk-actions-dropdown/bulk-actions-dropdown.component.html

```diff
@@ -0,0 +1,39 @@
+<mat-form-field
+  *ngIf="items?.length"
+  appearance="outline"
+  class="aca-bulk-form-field"
+>
+  <mat-select
+    *ngIf="(totalItems$ | async) as totalItems; else disabledMode"
+    [placeholder]="'SEARCH.BULK_ACTIONS_DROPDOWN' | translate: { count: totalItems }"
+    [title]="'SEARCH.BULK_ACTIONS_DROPDOWN' | translate: { count: totalItems }"
+    [disabled]="false"
+    panelClass="aca-bulk-select"
+    disableOptionCentering
+    data-automation-id="aca-bulk-dropdown"
+  >
+    <mat-option
+      *ngFor="let option of items;"
+      [value]="option.id"
+      [title]="option.description | translate"
+      [attr.data-automation-id]="option.id"
+    >
+      <adf-icon
+        *ngIf="option.icon"
+        [title]="option.title| translate"
+        [value]="option.icon"
+        [attr.data-automation-id]="'aca-option-icon-' + option.id"
+      ></adf-icon>
+      {{ option.title | translate }}
+    </mat-option>
+  </mat-select>
+
+<ng-template #disabledMode>
+  <mat-select
+    [placeholder]="'GOVERNANCE.MANAGE_HOLDS.BULK_NOT_AVAILABLE' | translate"
```

Comment:
I don't see these keys in the translation files. Are these referring to the ones created in the [ADW PR](https://github.com/Alfresco/alfresco-applications/pull/661/files#diff-43770a9fe54da59b46d103e9c4d007c2991f5b5f422ae3a4b7dbf5692eecce12R156)

Reply:
translations in ADW

Reply:
This won't work if someone will use plain ACA without ADW, this translation should be placed in ACA

Reply:
moved translations to ACA